src/styles/block/card.scss

```diff
@@ -0,0 +1,100 @@
+.card__button {
+  display: flex;
+  justify-content: center;
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-align: center;
+  color: rgb(255, 255, 255);
+  align-self: center;
```

Comment:
useless

Reply:
line-height: 16px; and color: rgb(255, 255, 255); I can't delete, change the color  and height of the word.
I deleted text-align: center; and  align-self: center;